src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,28 @@
+.calendar {
+  padding: 10px;
+  width: #{$limit-width-of-row};
+  display: flex;
+  gap: $indent-of-days;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    text-decoration: none;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+
+    align-items: center;
+    justify-content: center;
+    transition: 500ms;
+    display: none;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transform: translateY(-20px);
+  }
```

Comment:
The ':hover' selector is used to select elements when you mouse over them. But here, you're using it with elements that have 'display: none'. It won't have any effect because these elements are not displayed. You need to reconsider the way you're hiding and showing elements.